src/ensembl/src/shared/components/layout/sidebar-modal/SidebarModal.scss

```diff
@@ -0,0 +1,31 @@
+@import 'src/styles/common';
+
+.wrapper {
+  position: relative;
+  width: 100%;
```

Comment:
this is the default behaviour of display: grid; also unnecessary

Reply:
Removed it